### PR TITLE
openstack: Add Instructions to Run Installer

### PR DESCRIPTION
**Pull request by iamemilio**
+ also includes response to many nits from various reviewers

### CODE REVIEW

**Review by Fedosin — APPROVED** [review]
Thanks Emilio

File: docs/user/openstack/README.md [hunk 1]
```diff
@@ -264,9 +365,9 @@ You could also specify a specific IP address with /32 if you wish.
 
 You can verify the operation of the load balancer now if you wish, using the curl commands given below.
 
-Now the DNS entry for <cluster name>-api.<base domain> needs to be updated to point to the new load balancer:
+Now the DNS entry for `<cluster name>-api.<base domain>` needs to be updated to point to the new load balancer:
 
-```
+```txt
 <load balancer ip> <cluster-name>-api.<base domain>
```
Comment (racedo) [inline docs/user/openstack/README.md:371]:
```suggestion
curl <load balancer ip> <cluster-name>-api.<base domain>
```

File: docs/user/openstack/README.md [hunk 2]
```diff
@@ -39,60 +81,11 @@ The file can contain information about several clouds. For instance, the example
...
-     * Floating IPs: 3
-     * Security Groups: 3
-     * Security Group Rules: 60
-     * Routers: 1
-     * Subnets: 1
-     * RAM: 112 Gb
-     * VCPU: 28
-     * Volume Storage: 175 Gb
-     * Instances: 7
+## RHCOS Image
```
Comment (racedo) [inline docs/user/openstack/README.md:86]:
```suggestion
## Red Hat Enterprise Linux CoreOS (RHCOS)
```

File: docs/user/openstack/README.md [hunk 3]
```diff
@@ -39,60 +81,11 @@ The file can contain information about several clouds. For instance, the example
...
-```sh
-openstack quota set --secgroups 8 --secgroup-rules 100 <project>`
-```
-
-### RHCOS Image
-
-If you do not have a Red Hat Core OS image already, or are looking for the latest, [click here](https://mirror.openshift.com/pub/openshift-v4/dependencies/rhcos/pre-release/latest/).
-
-The installer requires a proper RHCOS image in the OpenStack cluster or project:
+If you do not have a Red Hat Core OS image already, get the latest image [here](https://mirror.openshift.com/pub/openshift-v4/dependencies/rhcos/pre-release/latest/). The installer requires a proper RHCOS image in the OpenStack cluster or project:
```
Comment (racedo) [inline docs/user/openstack/README.md:88]:
```suggestion
Get the latest RHCOS image [here](https://mirror.openshift.com/pub/openshift-v4/dependencies/rhcos/pre-release/latest/). The installer requires a proper RHCOS image in the OpenStack cluster or project:
```

**Review by racedo — CHANGES_REQUESTED** [review]

File: docs/user/openstack/README.md (hunk 1 above)
Comment (iamemilio) [reply docs/user/openstack/README.md:371]: This is a change made to dns

Comment (iamemilio) [reply docs/user/openstack/README.md:371]: I think this one is actually correct the way it is

**Pull request merged**